packages/strapi-plugin-upload/config/queries/mongoose.js

```diff
@@ -90,8 +90,8 @@ module.exports = {
     return this
       .find({
         '$or': [
-          { username: re },
-          { email: re }
+          { hash: re },
```

Comment:
You're not using the `params.id` value here?

Reply:
You don't see previous lines, `re` is a regular expression created with `params.id`